internal/repository/kyma/status/kyma_status_repository.go

```diff
@@ -0,0 +1,56 @@
+package status
```

Comment:
We usually use a short for `repo` in the file names: `internal/repository/kyma/status/kyma_status_repo.go`